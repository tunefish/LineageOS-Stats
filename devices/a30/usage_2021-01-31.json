{
  "17.1": 28,
  "18.0": 607,
  "18.1": 139,
  "none": 4
}