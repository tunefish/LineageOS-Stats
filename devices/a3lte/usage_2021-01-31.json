{
  "15.1": 507,
  "16.0": 180,
  "17.1": 587
}