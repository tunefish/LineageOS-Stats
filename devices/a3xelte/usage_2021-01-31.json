{
  "15.1": 101,
  "16.0": 434,
  "17.1": 3115,
  "18.0": 3,
  "18.1": 11,
  "52.8": 1,
  "none": 2
}