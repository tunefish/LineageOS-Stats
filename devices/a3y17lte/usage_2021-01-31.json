{
  "14.1": 5389,
  "15.1": 300,
  "16.0": 917,
  "17.1": 849,
  "18.0": 1
}