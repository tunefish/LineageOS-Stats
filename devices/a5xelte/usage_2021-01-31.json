{
  "15.1": 50,
  "16.0": 494,
  "17.1": 3746,
  "18.0": 12,
  "18.1": 1,
  "none": 1
}