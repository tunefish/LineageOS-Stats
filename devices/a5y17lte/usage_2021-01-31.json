{
  "14.1": 204,
  "15.1": 126,
  "16.0": 717,
  "17.1": 4146,
  "none": 1
}