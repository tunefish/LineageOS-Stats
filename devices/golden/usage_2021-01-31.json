{
  "13.0": 952,
  "14.1": 3020,
  "15.1": 4,
  "17.1": 3,
  "none": 1
}